addons/web/static/src/core/assets.js

```diff
@@ -83,19 +79,20 @@ assets.loadCSS = memoize(function loadCSS(url, retryCount = 0) {
             } else {
                 reject(new AssetsLoadingError(`The loading of ${url} failed`));
             }
-        });
+        };
     });
+    cachePromises.set(url, promise);
     document.head.appendChild(linkEl);
     return promise;
-});
+};
 
 /**
  * Get the files information as descriptor object from a public asset template.
  *
  * @param {string} bundleName Name of the bundle containing the list of files
  * @returns {Promise<{cssLibs, cssContents, jsLibs, jsContents}>}
  */
-assets.getBundle = memoize(async function getBundle(bundleName) {
+assets.getBundle = async function getBundle(bundleName) {
```

Comment:
Why do you remove memoize here?

Reply:
the goal is to use a cache (here const cacheMap = new Map()) and so to delete memoize everywhere. 
I added cache for getBundle()